parm/config/config.coupled_ic

```diff
@@ -8,7 +8,8 @@ echo "BEGIN: config.coupled_ic"
 source ${EXPDIR}/config.resources coupled_ic
 
 if [[ "${machine}" == "WCOSS2" ]]; then
-  export BASE_CPLIC="/lfs/h2/emc/global/noscrub/emc.global/IC/COUPLED"
+  #export BASE_CPLIC="/lfs/h2/emc/global/noscrub/emc.global/IC/COUPLED"
+  export BASE_CPLIC="/lfs/h2/emc/couple/noscrub/Jiande.Wang/IC"
```

Comment:
Does Jiande have the old prototype ICs there too? I'd be best to keep them all in one place.

Reply:
@jiandewang is copying the 20130401 case so that testing can run seamlessly.  I don't know if there's a need for the C384 P8 ICs on WCOSS2 beyond that.

Reply:
You can remove the old line then and I'll delete our copy to free up space.

Reply:
Will do.  That sounds like a good plan.  Can we wait to confirm with @jiandewang that he's copied over (so he doesn't have to do from another machine) before you delete on wcoss2?

Reply:
> Will do. That sounds like a good plan. Can we wait to confirm with @jiandewang that he's copied over (so he doesn't have to do from another machine) before you delete on wcoss2?

that's a 7T dir, needs time to do the copy, will signal you by the time

Reply:
> > Will do. That sounds like a good plan. Can we wait to confirm with @jiandewang that he's copied over (so he doesn't have to do from another machine) before you delete on wcoss2?
> 
> that's a 7T dir, needs time to do the copy, will signal you by the time

need chmod from global accoun holder for the following
-rw-r-----   1 emc.global global  2422 Aug 15  2022 copyWW3.20210528v2.sh
-rw-r-----   1 emc.global global  2429 May 31  2022 copyWW3ic.sh

Reply:
/lfs/h2/emc/global/noscrub/emc.global/IC/COUPLED on dogwood are all copied under /lfs/h2/emc/couple/noscrub/Jiande.Wang/IC except for three *sh files that needs to have chmod

Reply:
Thanks @jiandewang  --  @WalterKolczynski-NOAA  the commented out line is now removed.